docs/apis/streaming/connectors/kinesis.md

```diff
@@ -299,3 +299,41 @@ Otherwise, the returned stream name is used.
 Other optional configuration keys can be found in `KinesisConfigConstants`.
 		
 		
+### Using non-AWS Kinesis Endpoints
```

Comment:
"N"on-AWS. Also, I think it'd be good to let the title be "Using Non-AWS Kinesis Endpoints for Testing", to better point out the purpose of doing this.